account-kit/wallet-client/src/exports/internal.ts

```diff
@@ -0,0 +1,5 @@
+// exports from this file are meant to be imported by our packages in the monorepo
+// we will expose this in the package.json as a named export like `@account-kit/wallet-client/internal`
+export * from "../internal/account.js";
```

Comment:
debating if we should keep this exported or if it's light enough now to just copy over & not share w/ the server (see https://github.com/alchemyplatform/wallet-server/pull/258#discussion_r2193168097)